tests/operator_test.go

```diff
@@ -97,6 +100,7 @@ var _ = Describe("Operator tests", func() {
 
 var _ = Describe("Operator delete CDI tests", func() {
 	var cr *cdiv1.CDI
+	var cdiOperator *appsv1.Deployment
```

Comment:
Can we call this cdiOperatorDeployment, when reading the code I keep thinking this is the pod, but it is actually the deployment.

Reply:
Thanks, changed.